### DISCUSSION TITLE
How to use this for pre - rendering ssr

### DISCUSSION THREAD

**Discussion by todaysprint** [hoangvvo/nextjs-mongodb-app#151] [Q&A]
Hi First Thanks for good Script 
iam looking for fetch with pre-rending through getServerSideProps 
how can it done please reply thanks

**Comment by danielmeeusen** [comment]
Could you be a little more specific as to what you are looking for?

There is an example of using getServerSideProps to load data at the bottom of [postId].jsx

Here: https://github.com/hoangvvo/nextjs-mongodb-app/blob/v2/pages/user/%5Busername%5D/post/%5BpostId%5D.jsx
